galleon-plugins/src/main/java/org/wildfly/galleon/plugin/WfInstallPlugin.java

```diff
@@ -97,6 +97,7 @@
 import org.wildfly.galleon.plugin.config.DeletePath;
 import org.wildfly.galleon.plugin.config.ExampleFpConfigs;
 import org.wildfly.galleon.plugin.config.FileFilter;
+import org.wildfly.galleon.plugin.config.FileAppender;
```

Comment:
Now that you moved the method to the task impl, please reset changes in this class.